server/tests/model/users.spec.js

```diff
@@ -0,0 +1,67 @@
+import chai from "chai";
+import chaiAsPromise from "chai-as-promised";
+import models from "../../models";
+import { userData } from "../fixtures/model/userData";
+
+chai.use(chaiAsPromise);
+const sequelize = models.sequelize;
+const assert = chai.assert;
+const { Users } = models;
+
+beforeEach(async () => {
+  await sequelize.sync({ force: true }).catch(() => {});
+});
+
+describe("User table Model", async () => {
+  it("should create a User model", async () => {
+    const createdUser = await Users.create(userData);
+    assert.equal(
+      createdUser.dataValues.firstName,
+      userData.firstName,
+      "firstName"
+    );
+    assert.equal(createdUser.dataValues.lastName, userData.lastName, "last");
+    assert.equal(createdUser.dataValues.email, userData.email, "email");
+    assert.equal(createdUser.dataValues.gender, userData.gender, "gender");
+    assert.equal(
+      createdUser.dataValues.biography,
+      userData.biography,
+      "biography"
+    );
+    assert.equal(
+      createdUser.dataValues.password,
+      userData.password,
+      "password"
+    );
+    assert.equal(
+      createdUser.dataValues.mobileNumber,
+      userData.mobileNumber,
+      "mobileNumber"
+    );
+    assert.equal(
+      createdUser.dataValues.imageURL,
+      userData.imageURL,
+      "imageURL"
+    );
+    assert.equal(
+      createdUser.dataValues.isNotifiable,
+      userData.isNotifiable,
+      "isNotifiable"
+    );
+    assert.equal(createdUser.dataValues.isAdmin, userData.isAdmin, "isAdmin");
+  });
+
+  it("should delete a User model", async () => {
+    const createdUser = await Users.create(userData);
+    assert.equal(createdUser.dataValues.email, userData.email);
+    await Users.drop({ cascade: true });
+    chai
+      .expect(Users.findOne({ where: { email: userData.email } }))
+      .to.rejectedWith(Error);
+  });
+});
+
+after(async () => {
+  sequelize.drop();
+  sequelize.close();
```

Comment:
I don't think this is necessary since you're dropping the database before starting each test with `sequelize.sync({ force: true })`

Reply:
It is need for cleanup and to encourage stateless test through process, also given how mocha runs the test in non-sequential mode. You cant tell which test will run first so it is important to handle your test without depending on another test to do clean up for you.